include/sound/sof/ext_manifest4.h

```diff
@@ -90,6 +90,18 @@ struct sof_man4_segment_desc {
 	uint32_t file_offset;
 } __packed;
 
+/* bit definition for type in sof_man4_module */
+#define SOF_IPC4_MODULE_LOAD_TYPE_MASK 0xf
```

Comment:
The commit message is very very hard to correlate with the code changes, where it seems you moved one assignment. No idea what problem you are trying to fix. You are probably right but what was the problem again?

Reply:
thanks, updated